src/cache/cache.cljs

```diff
@@ -57,7 +57,9 @@
       (remote-synchronized/create (get-storage (remote-synchronized/extract-local-config config))
                                   (get-storage (remote-synchronized/extract-remote-config config)))
 
-      (file/create config))))
+      (if (not (empty? (:server (:backend_options config))))
```

Comment:
Even though it is very common in PHP, using `(not (empty? ...))` is not considered idiomatic in Clojure.  
Instead, using [`seq`](https://clojuredocs.org/clojure.core/seq) is recommended (it turns any empty collection type into `nil`, which includes empty strings).

Could you please change it to
```
      (if (seq (:server (:backend_options config)))
        (redis/create config)
        (file/create config)))))
```

Reply:
Sure, fixed!

Reply:
Almost - `(seq (:server (:backend_options config))` is `nil` if there is no backend_options server, so the `(file/create config)` option needs to be first, the `(redis/create config)` second.

Reply:
Please ignore my previous comment that I just deleted - I wasn't quite awake yet.